docs/weighting/transitionMethodology_en.md

```diff
@@ -0,0 +1,32 @@
+# Methodology Used in Transition for Weighting
+
+## Weighting of Stop Nodes and Paths
+
+Transition allows the user to choose the decay formula and select either distance or travel time as the input variable for the [formula](poiAndTransitNodeWeighting_en.md#distance-and-travel-time-decay-functions).
+
+### Weighting Steps and Specifications
+
+- Creation in Transition or importation of stop nodes from a GTFS file
```

Comment:
Might be related to my previous comment, but I don't understand what we are trying to say here.